site/pages/getting-started/code/instrumentation.tsx

```diff
@@ -46,7 +47,12 @@ const customEventRows = [
 ];
 
 const contentOverrides = {
-  typographyPreset: 'editorialParagraph030',
+  typographyPreset: {
+    xs: 'editorialParagraph020',
+    md: 'editorialParagraph030',
+    lg: 'editorialParagraph030',
+    xl: 'editorialParagraph030',
+  },
```

Comment:
you can do like that to keep it shorter:
```suggestion
  typographyPreset: {
    xs: 'editorialParagraph020',
    md: 'editorialParagraph030',
  },
```